### PR TITLE
Airflow taks tests

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: resource/airflow_dag_pipeline.yaml [hunk 1]
```diff
@@ -174,7 +174,7 @@ jobs:
             - |
               cp -R ./pr-with-iam/* ./pull-request-edit
               jq '.role_name = "airflow_test_dag_role"' ./pr-with-iam/deploy.json > ./pull-request-edit/deploy.json
-              jq '.Statement += [{"Sid": "denyall","Action": ["*"],"Effect": "Deny","Resource": ["*"]}]' ./pr-with-iam/iam_policy.json > ./pull-request-edit/iam_policy.json
```
Comment (xoen) [inline resource/airflow_dag_pipeline.yaml:177]: What was the problem with the Sid in this statement?

File: resource/airflow_dag_pipeline.yaml [hunk 2]
```diff
@@ -192,6 +192,7 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ((secrets.role-putter-secret-access-key))
         AWS_NODES_ROLE_ARN: ((secrets.nodes-role-arn))
         APP_NAME: ((app-name))
+        DELETE_AFTER_TEST: true
```
Comment (xoen) [inline resource/airflow_dag_pipeline.yaml:195]: How/Where is this used?

File: resource/airflow_dag_pipeline.yaml (hunk 1 above)
Comment (s-block) [reply resource/airflow_dag_pipeline.yaml:177]: If someone has a policy with another sid of the same name it fails. Best to have no sid so there will not be conflicts

**Review by xoen — APPROVED** [review]
As far as I can tell and with the little I know about this, it looks OK to me.

File: resource/airflow_dag_pipeline.yaml (hunk 2 above)
Comment (s-block) [reply resource/airflow_dag_pipeline.yaml:195]: In `put-iam-policy` this is used to delete the policy if it was created when testing with a `Deny` all statement appended.

File: resource/airflow_dag_pipeline.yaml (hunk 1 above)
Comment (xoen) [reply resource/airflow_dag_pipeline.yaml:177]: 👍

**Pull request merged**